client/tasks/task-list-item.tsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import { __, _n } from '@wordpress/i18n';
+import {
+	getHistory,
+	getNewPath,
+	updateQueryString,
+} from '@woocommerce/navigation';
+import { ONBOARDING_STORE_NAME, useUserPreferences } from '@woocommerce/data';
+import { recordEvent } from '@woocommerce/tracks';
+import { TaskItem } from '@woocommerce/experimental';
+import { useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { TaskProps } from './task-list';
+import './task-list.scss';
+
+export type TaskListItemProps = {
+	isExpandable: boolean;
+	isExpanded: boolean;
+	setExpandedTask: ( id: string ) => void;
+	task: TaskProps;
+};
+
+export const TaskListItem: React.FC< TaskListItemProps > = ( {
+	isExpandable = false,
+	isExpanded = false,
+	setExpandedTask,
+	task,
+} ) => {
+	const { createNotice } = useDispatch( 'core/notices' );
+	const {
+		dismissTask,
+		snoozeTask,
+		undoDismissTask,
+		undoSnoozeTask,
+	} = useDispatch( ONBOARDING_STORE_NAME );
+	const userPreferences = useUserPreferences();
+
+	const {
+		actionLabel,
+		actionUrl,
+		content,
+		id,
+		isComplete,
+		isDismissable,
+		isSnoozable,
+		time,
+		title,
+	} = task;
+
+	const onDismiss = () => {
```

Comment:
Might want to wrap this in `useCallback` since it's fed to another component, to avoid excessive rerenders.

Reply:
Good call(back).  Done in https://github.com/woocommerce/woocommerce-admin/pull/7556/commits/48219bc7b6cab5bd833bd9bee9f53dd0e77230f4